### PR TITLE
Trace prepared statement name

### PR DESCRIPTION
**Pull request by lzakharov**
Add prepared statement name to the trace attributes.

### CODE REVIEW

**Review by obitech — APPROVED** [review]
LGTM

File: tracer.go [hunk 1]
```diff
@@ -24,6 +24,8 @@ const (
 	QueryParametersKey = attribute.Key("pgx.query.parameters")
 	// BatchSizeKey represents the batch size.
 	BatchSizeKey = attribute.Key("pgx.batch.size")
+	// PrepareStmtNameKey represents the prepared statement name.
+	PrepareStmtNameKey = attribute.Key("pgx.prepare_stmt.name")
```
Comment (costela) [inline tracer.go:28]: not a blocker for this MR, but this feels like something that could be added to semconv: https://github.com/open-telemetry/semantic-conventions/blob/9790d8f54533840dd187d3ffa5c2546d9420d357/docs/database/sql.md

File: tracer.go [hunk 2]
```diff
@@ -304,6 +306,7 @@ func (t *Tracer) TracePrepareStart(ctx context.Context, conn *pgx.Conn, data pgx
 	opts := []trace.SpanStartOption{
 		trace.WithSpanKind(trace.SpanKindClient),
 		trace.WithAttributes(t.attrs...),
+		trace.WithAttributes(PrepareStmtNameKey.String(data.Name)),
 	}
```
Comment (costela) [inline tracer.go:310]: minor: AFAIK we still can have non-prepared statements, so maybe we could only add this if the statement has a name?

```suggestion
	}
	
	if data.Name != "" {
		trace.WithAttributes(PrepareStmtNameKey.String(data.Name)),
	}
```

**Review by costela — CHANGES_REQUESTED** [review]
thanks for the contribution! :rocket:

**Review by costela — APPROVED** [review]
thanks again! :pray:

**Pull request merged**